lib/plugins/aws/utils/formatLambdaLogEvent.js

```diff
@@ -8,17 +8,27 @@ const { style } = require('@serverless/utils/log');
 module.exports = (msgParam, options = {}) => {
   const { isModern } = options;
   let msg = msgParam;
-  const dateFormat = 'YYYY-MM-DD HH:mm:ss.SSS (Z)';
```

Comment:
Quick question: is the date formatted in the local timezone (the one of the computer)?

(if yes, perfect)

Reply:
yes, that's the case

Reply:
Thanks!